tests/ext/sandbox-prehook/exit_and_drop_span.phpt

```diff
@@ -0,0 +1,23 @@
+--TEST--
+[Prehook Regression] Exit gracefully handles a dropped span
```

Comment:
This doesn't actually test that the span gets dropped though, does it?

Reply:
No, that's tested with `tests/ext/sandbox-prehook/drop_spans.phpt`. This test makes sure that a dropped span with `exit` does not segfault (aded in #765).